src/TypealizR/StringLocalizerExtensionsSourceGenerator/ExtensionClassModel.cs

```diff
@@ -42,16 +42,16 @@ namespace Microsoft.Extensions.Localization
 
             {{generatorType.GeneratedCodeAttribute()}}
             /// <summary>
-            /// Extensions for <see cref="{{markertType.FullName}}" to utilize typed-access to ressources generated by TypealizR />
+            /// Extensions for <see cref="{{markertType.FullName}}"/> to utilize typed-access to ressources generated by TypealizR.
             /// </summary>
             {{markertType.Accessibility.ToVisibilty().ToString().ToLowerInvariant()}} static partial class IStringLocalizerExtensions{{markertType.FullNameForClassName}}
             {
 
                 {{methods.Select(x => x.ToCSharp()).ToMultiline()}}
 
                 /// <summary>
-                /// wraps the specified <see cref="IStringLocalizer{{{markertType.FullName}}}"/> into a generated type providing properties to access [Some.Nested.Group]: via properties
-                /// IStringLocalizer<{{markertType.FullName}}> localize = ...
+                /// wraps the specified <see cref="IStringLocalizer{{markertType.Name}}"/> into a generated type providing properties to access [Some.Nested.Group]: via properties

```

Comment:
@nielsNocore 

Not quite sure about this, as this would result in `IStringLocalizer{T}`, instead of the more discrete `IStringLocalizer{DiscreteTypeNameGoesHere}`, which is also used in the generated constructor. Can leave it as is, for now, but might change it back